not-so-smart-contracts/algorand/access_controls/README.md

```diff
@@ -1,18 +1,17 @@
 # Access Controls
 
-Lack of appropriate checks for application calls of type UpdateApplication and DeleteApplication allows attackers to update application’s code or delete an application entirely.
+Inadequate checks for application calls of type UpdateApplication and DeleteApplication can allow attackers to alter an application's code or delete it entirely.
```

Comment:
We should probably stick with previously used `update` instead of `alter an` (?)